src/containers/widget/index.js

```diff
@@ -81,6 +101,24 @@ const Widget = ({
             onNextClick={handleDonateNextClick}
           />
         );
+      case "payment-screen":
+        return (
+          <PaymentScreen
+            title={widgetTitle}
+            currency={currency}
+            onRequestClose={closeModal}
+            selectedAmount={selectedAmount}
+            {...invoiceDetails}
```

Comment:
the details might change? 
does it make sense to pass in the invoice as object instead of each attribute individual?

Reply:
If the keys won't change it won't be a problem, I am just spreading al the props instead of passing each key.

Reply:
any objections in passing the invoice object? 
the API response for the invoiceDetails might change.